cmd/manager/main.go

```diff
@@ -189,3 +207,22 @@ func serveCRMetrics(cfg *rest.Config) error {
 	}
 	return nil
 }
+
+// checkIfRequiredFilesExist checks for the existence of required files and binaries before starting WMCO
+// sample error message: errors encountered with required files: could not stat /payload/hybrid-overlay.exe:
+// stat /payload/hybrid-overlay.exe: no such file or directory, could not stat /payload/wmcb.exe: stat /payload/wmcb.exe:
+// no such file or directory
+func checkIfRequiredFilesExist(requiredFiles []string) error {
+	var errors []string
```

Comment:
errors with string slice sounds weird.

Reply:
But that is what the slice of strings contains.

Reply:
nit again: Perhaps `errstrings` or `errorstrings` as errors might make it confusing to ppl reading the code.

Reply:
That goes against go naming conventions. Maybe `errorMessages` or `errMsgs`.

Reply:
Ya, I was just giving an example but I realized it was a wrong example.

Reply:
Went ahead with `errorMessages`